Makefile

```
@@ -4,7 +4,7 @@ SHELL = /bin/bash
 # This value must be updated to the release tag of the most recent release, a change that must
 # occur in the release commit. IMAGE_VERSION will be removed once each subproject that uses this
 # version is moved to a separate repo and release process.
-export IMAGE_VERSION = v1.8.1
+export IMAGE_VERSION = 1.8.2
```

```suggestion
export IMAGE_VERSION = v1.8.2
```
I believe that will fix the broken link as well.

---

The same replacement (1.8.2 --> v1.8.2) needs to happen across all files.